packages/frontend-2/lib/auth/composables/onboarding.ts

```diff
@@ -101,6 +101,12 @@ export const useProcessOnboarding = () => {
     const user = activeUser.value
     if (!user) throw new OnboardingError('Attempting to onboard unidentified user')
 
+    mixpanel.people.set_once({
+      onboardingSurveyRole: onboardingData?.role,
+      onboardingSurveyPlans: onboardingData?.plans,
```

Comment:
Some of these answers are very long, like "Business intelligence (data analytics and visualisation)". Maybe we just create an unique key for each of them that we use rather than the actual text?

Reply:
Nvm these are actually already the unique keys